src/query/query_executor.c

```diff
@@ -12343,29 +12346,46 @@ qexec_execute_selupd_list (THREAD_ENTRY * thread_p, XASL_NODE * xasl, XASL_STATE
   REGU_VARLIST_LIST outptr = NULL;
   REGU_VARIABLE *varptr = NULL;
   DB_VALUE *rightvalp = NULL, *thirdvalp = NULL;
-  CL_ATTR_ID attrid;
-  int n_increment;
-  int savepoint_used = 0;
-  OID *oid = NULL, *class_oid = NULL, class_oid_buf, last_cached_class_oid;
-  HFID *class_hfid = NULL;
+  bool savepoint_used = false;
```

Comment:
It was used because we could theoretically have a case when savepoint_used was true and submvcc_used was not. Removed error code from logtb_assign_subtransaction_mvccid and submvcc_used.